src/app/modules/main/stickers/module.nim

```diff
@@ -163,6 +160,21 @@ method estimate*(self: Module, packId: string, address: string, price: string, u
 method addRecentStickerToList*(self: Module, sticker: StickerDto) =
   self.view.addRecentStickerToList(initItem(sticker.hash, sticker.packId, sticker.url))
 
+method getRecentStickers*(self: Module) =
+  let data = self.controller.getRecentStickers()
+  if data.len > 0:
+    for sticker in data:
+      self.addRecentStickerToList(sticker)
+      return
+  self.controller.loadRecentStickers()
+
+method getInstalledStickerPacks*(self: Module) =
```

Comment:
We can make those two functions smarter and check the model if the sticker packs and recent stickers are already loaded (we can use a flag if we want too), so that we ever just fetch once, what do you think?

Reply:
On it